src/components/Sponsors.astro

```diff
@@ -101,4 +101,9 @@ const logoList: Array<{
       )
     }
   </div>
+  <div
+    class="flex w-full items-center justify-center p-4 text-lg text-base-content"
+  >
+    and more!
```

Comment:
The text colour for this is unreadable, let's change it to white like other pieces of text on the page.

Reply:
Fixed!